src/game.py

```diff
@@ -107,7 +107,7 @@ def initialize():
     
     #Generate Level
     level = level_generator.generate_level(load_path = 
-                                           'assets/sample_audio_files/break_free_cut.ogg',
+                                           'assets/sample_audio_files/laugh_now_cry_later.ogg',
                                            save_path='assets/level.npy',
                                            min_onset_strength=0.3,
                                            min_onset_gap=0.75)
```

Comment:
Once we have fine-tuned the arguments, I think we should set these variables in level_generator to constants for simplicity's sake.